node_modules/col-core/lib/api.js

```diff
@@ -193,6 +194,96 @@ var initializeAuthorizationMiddleware = function(apiRouter) {
   });
 };
 
+
+/**
+ * Generate previews for an asset
+ *
+ * @param  {Asset}        asset             The asset for which previews are generated
+ * @param  {Function}     callback          Standard callback function
+ * @param  {Object}       callback.err      An error object, if any
+ */
+var generatePreviews = module.exports.generatePreviews = function(id, url, postBackEndpoint, callback) {
```

Comment:
Why are we moving this into `col-core`? Wouldn't we want this in the asset API ?